pypit/arpca.py

```diff
@@ -123,10 +126,7 @@ def basis(xfit, yfit, coeff, npc, pnpc, weights=None, skipx0=True, x0in=None, ma
     x3fit = np.dot(eigv,high_order_matrix.T) + np.outer(x0fit,np.ones(nrow)).T
     outpar = dict({'high_fit': high_fit, 'x0': x0, 'x0in': x0in, 'x0fit': x0fit, 'x0res': x0res, 'x0mask': fitmask,
                    'hidden': hidden, 'usetrc': usetrace, 'eigv': eigv, 'npc': npc, 'coeffstr': coeffstr})
-    if retmask:
-        return x3fit, outpar, tmask
-    else:
-        return x3fit, outpar
+    return x3fit, outpar
```

Comment:
Are you sure that retmask option is no longer needed?

Just checking..

Reply:
It doesn't seem to be called by any other function in the code, so I imagine this is now deprecated.